docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,43 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+## Upgrade from a version prior to v1.13
+
+Prior to Antrea v1.13, the `ClusterClaim` CRD is used to define both the Cluster ID and
+ClusterSet ID for a ClusterSet. Since Antrea v1.13, the `ClusterClaim` CRD is removed, and
+the `ClusterSet` CRD solely defines a ClusterSet. The name of a ClusterSet CR must match the
+ClusterSet ID, and a new `clusterID` field specifies the Cluster ID.
+
+To upgrade from a version older than v1.13, once you apply the latest manifests for an
```

Comment:
We should talk about the new version MC Controller will still recognize old version ClusterClaim and ClusterSet CRs, but if the ClusterSet name does not match the ClusterSet ID, you should delete the ClusterSet CR and create a new one with using ClusterSet ID as the name.

Reply:
I found that the old codes already validate the ClusterSet ID and the ClusterSet CR's name to make sure they are the same, so I suppose there is no need to handle the recreate case.  Both leader and member ClusterSet controller will call `ValidateLocalClusterClaim`: https://github.com/antrea-io/antrea/blob/1e0fbd6d367d9c556928bacf1bac6ce89a4e596e/multicluster/controllers/multicluster/member/clusterset_controller.go#L116

The method will check if the ClusterSet ID and ClusterSet's name matched.
https://github.com/antrea-io/antrea/blob/1e0fbd6d367d9c556928bacf1bac6ce89a4e596e/multicluster/controllers/multicluster/common/controller_utils.go#L74-L78

Reply:
Ok. This is good to know.